dev/io.openliberty.data.internal_fat_errorpaths/test-applications/DataErrPathsTestApp/src/test/jakarta/data/errpaths/web/Invention.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package test.jakarta.data.errpaths.web;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+
+/**
+ * An invalid entity - because it lacks an Id attribute.
+ */
+@Entity
+public class Invention {
+
```

Comment:
Issue: The test claims that this is an invalid entity because it contains two `@Id` attribute without the use of `@IdClass`.  However, this entity is invalid because it lacks an Id attribute.

Either update the test description, or add two id attributes to this entity.

Reply:
Thanks for spotting that! I corrected the test description in https://github.com/OpenLiberty/open-liberty/pull/30062/commits/5b97a93c7fba168126370efa43a916fc5727d279